Gemfile

```diff
@@ -0,0 +1,6 @@
+Encoding.default_external = Encoding::UTF_8
```

Comment:
hm why this file added?

Reply:
please see : 
https://github.com/kubevirt/user-guide/pull/285
It did not passed CI without it...

Reply:
@alonSadan the #285 has been merged, can you rebase on top of current master?

 Thanks!